benchmark/build.gradle.kts

```diff
@@ -57,3 +62,12 @@ dependencies {
     // https://developer.android.com/studio/projects/android-library#Convert
     androidTestImplementation(project(":AutomatticTracks"))
 }
+
+dependencyAnalysis {
+    issues {
+        onUnusedDependencies {
+            // This dependency is not needed but kept to preserve the module's default configuration.
+            exclude("org.jetbrains.kotlin:kotlin-stdlib")
+        }
+    }
+}
```

Comment:
👋 @wzieba !

> ...then let's maybe instruct kotlin gradle plugin to... not include kotlin-stdlib.

To my understanding, you suggest adding the `kotlin.stdlib.default.dependency=false` within `gradle.properties`, but have you (maybe) tried that yourself? This, as I would also expect, fails the build, because although this `benchmark` module doesn't need the `kotlin-stdlib` dependency, all other modules do. Isn't that right? 🤔

What we want is to tell our build to ignore the `kotlin-stdlib` dependency, yes, but for `benchmark` only. I am not sure what's the correct way to do that, especially since this also needs to be done on the source-set level. 🤷 

PS: I think that for what is worth we shouldn't care too much about the configuration of this module, it is just an auto-generate benchmark module that serve a very specific purpose.

Reply:
Each module can have its own `gradle.properties`. Having `kotlin.stdlib.default.dependency=false` in the `gradle.properties` in `benchmark` works just fine

Reply:
True @wzieba , I just tried it and it indeed works, I am curious why though, as I would have expected that `./gradlew benchmark:assembleAndroidTest` to fail since `TrackingMethodsBenchmark` is written in Kotlin, but it works and the project compiles, strange... 🤔 

No matter, this is now done: a2a349dff6166c6d9fe56b200fa80a7ff02df7b6

Reply:
Thanks! Just sharing that the stdlib contains some additional features, but it's not necessary to compile Kotlin code - that's why this module still compiles.

Reply:
Ah true @wzieba (😅 ) , it is just a Kotlin standard library with some additional features... I didn't have to deal with that for so long that I am now forgetting things, thanks for the merge! 🤷